python_modules/dagster/dagster_tests/definitions_tests/auto_materialize_tests/updated_scenarios/custom_condition_scenarios.py

```diff
@@ -0,0 +1,25 @@
+from dagster import AutoMaterializeRule
+from dagster._core.definitions.asset_condition import RuleCondition
+from dagster._core.definitions.auto_materialize_policy import AutoMaterializePolicy
+
+from ..asset_daemon_scenario import (
+    AssetDaemonScenario,
+)
+from ..base_scenario import run_request
+from .asset_daemon_scenario_states import (
+    one_asset,
+)
+
+custom_condition_scenarios = [
+    AssetDaemonScenario(
+        id="lotta_nots_scenario",
+        initial_state=one_asset.with_asset_properties(
+            auto_materialize_policy=AutoMaterializePolicy.from_asset_condition(
+                ~~~~~~~~~~RuleCondition(AutoMaterializeRule.materialize_on_missing())
```

Comment:
`~~~~~~~~~~` ?

Reply:
Ok that could have been explained better haha -- it's basically just doing a bunch of `NotAssetCondition(NotAssetCondition(...))` just as an example of a thing that has a different structure than is currently allowed by the AutoMaterializeRule system. I'll update this test.

Reply:
got it, comment is probably sufficient (but necessary)